blocks/popup/popup.css

```
@@ -0,0 +1,38 @@
+.popup div[role="dialog"][aria-hidden="false"] {
+  position: fixed;
+  inset: 0;
+  margin: 0;
+  padding: 0;
+  background-color: var(--color-purple-transparent);
+  z-index: 1000;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

In theory, we only need te `display` prop to have higher specificity, so we could also do:
```suggestion
.popup div[role="dialog"] {
  position: fixed;
  inset: 0;
  margin: 0;
  padding: 0;
  background-color: var(--color-purple-transparent);
  z-index: 1000;
  justify-content: center;
  align-items: center;
}
.popup div[role="dialog"][aria-hidden="false"] {
  display: flex;
}
```